app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script.rb

```diff
@@ -33,4 +33,20 @@ def create_in_provider_queue(manager_id, params, auth_user = nil)
       MiqTask.generic_action_with_callback(task_opts, queue_opts)
     end
   end
+
+  def run(vars = {})
+    options = vars.merge(merge_extra_vars(vars[:extra_vars]))
+
+    with_provider_object do |jt|
+      jt.launch(options)
+    end
+  end
+
+  def merge_extra_vars(external)
+    {:extra_vars => variables.merge(external || {}).to_json}
+  end
+
+  def provider_object(connection = nil)
+    (connection || connection_source.connect).api.job_templates.find(manager_ref)
+  end
```

Comment:
Not added:

https://github.com/ManageIQ/manageiq/blob/8773af50d534452ad8d4b50c6ed3074f680e9fe5/app/models/manageiq/providers/ansible_tower/automation_manager/configuration_script.rb#L16